configs/multer_config.js

```diff
@@ -0,0 +1,30 @@
+const multer = require("multer");
+const path = require("path");
+const isImage = require("is-image");
+module.exports = multer({
+  storage: multer.diskStorage({
+    // extentions
+    filename: function (request, file, callback) {
+      const suff = Math.round(Math.random() * 1e9);
```

Comment:
crypto.pseudoRandomBytes()/Math.random() is a cryptographically weak random number generator.

https://stackoverflow.com/questions/5651789/is-math-random-cryptographically-secure

Reply:
Removed line 8 as it is not required in cloudinary

Reply:
> https://deepsource.io/gh/cu-coders/backend/run/0d1b636a-0c64-4135-a2cb-aa72ef887b3b/javascript/

fixed the necessary ones.